src/jwt.interceptor.ts

```diff
@@ -7,8 +7,7 @@ import {
 } from '@angular/common/http';
 import { JwtHelperService } from './jwthelper.service';
 import { JWT_OPTIONS } from './jwtoptions.token';
-import { Observable } from "rxjs/internal/Observable";
-import { from } from "rxjs/internal/observable/from";
+import { from, Observable } from "rxjs";
```

Comment:
Should have single quotes?

Reply:
Just reused the current style.

Reply:
Would be good to have single quotes, but I'm going to do a Prettier pass on it before the next release anyway.